setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
   name='codemon',
-  version='0.0.1',
+  version='0.0.2',
```

Comment:
Version is changed, to avoid confusion

Reply:
You can keep version changed locally for your personal use, please don't include it in the PR, since I don't want to update version of the package.

Reply:
Ok sure. 1 minute...